src/script/page/MainContent/panels/preferences/DevicesPreferences/components/E2EICertificateDetails/useCertificateDetailsModal.tsx

```diff
@@ -0,0 +1,120 @@
+/*
+ * Wire
+ * Copyright (C) 2023 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import {useState, useEffect} from 'react';
+
+import {PrimaryModal, removeCurrentModal} from 'Components/Modals/PrimaryModal';
+import {ModalOptions} from 'Components/Modals/PrimaryModal/PrimaryModalTypes';
+import {t} from 'Util/LocalizerUtil';
+import {downloadFile} from 'Util/util';
+
+const COPY_MESSAGE_TIMEOUT = 3000;
+const DOWNLOAD_CERTIFICATE_TIMEOUT = 500;
+const CERTIFICATE_NAME = 'certificate.pem';
+const CERTIFICATE_TYPE = 'application/x-pem-file';
+
+const ModalMessageWrapper = ({message}: {message: string}) => (
+  <div
+    style={{
+      overflow: 'auto',
+      maxHeight: '350px',
+      fontSize: 'var(--font-size-small)',
+      letterSpacing: '0.05px',
+      lineHeight: 'var(--line-height-md)',
+      wordBreak: 'break-word',
+      // The margin is set to -30px to compensate for the margin of the modal content
+      marginBottom: '-30px',
+    }}
+  >
+    <p>{message}</p>
+  </div>
+);
```

Comment:
Why are we getting rid of the `.style.ts` file?

Reply:
Was not needed in this case. I don't see a benefit in placing styles for this component in another file. This particular style is tailored to this particular component and, therefore, should be placed right next to (or into) it.